packages/flame_test/lib/src/test_flame_game.dart

```diff
@@ -26,9 +26,23 @@ import 'package:meta/meta.dart';
 @isTest
 Future<void> testWithFlameGame(
   String testName,
-  AsyncGameFunction<FlameGame> testBody,
-) {
-  return testWithGame<FlameGame>(testName, FlameGame.new, testBody);
+  AsyncGameFunction<FlameGame> testBody, {
+  Timeout? timeout,
+  dynamic tags,
+  dynamic skip,
```

Comment:
```suggestion
  Object? tags,
  Object? skip,
```

Reply:
Shouldn't we match the api from test?

Reply:
They just have it as `dynamic` for legacy reasons, but we could, doesn't matter that much.

Reply:
got ya, I slightly feel we would rather keep matching their apis. Gonna go with as is right now.